src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -166,11 +166,13 @@ def _resolve_model(self):
                 break
 
         doc_string = doc_string.replace('\r', '').replace('\n', ' ')
+        doc_string = re.sub(' +', ' ', doc_string)
```

Comment:
When "that returns" happens after the line break, the regex on line 170 fails due to the leading space on each line. Compressing multiple spaces to just one fixes it.